pki/certs.go

```diff
@@ -103,17 +103,18 @@ type CertificatePair struct {
 // to a log. The certificate's subject common name will be a random subdomain
 // based on the certificate serial under the `testCertDomain` domain.
 //
-// If windowStart is nil the certificate NotBefore will be set to the current
-// time based on the provided clock. If windowStart is not nil then the
-// certificate NotBefore will be set to the windowStart plus one day.
-
-// If windowEnd is nil the certificate NotAfter will be set to 90 days after the
-// current time based on the provided clock. If windowEnd is not nil then the
-// certificate NotAfter will be set to the windowEnd minus one day.
+// If windowEnd is nil, issue a 90 day certificate with validity starting today.
```

Comment:
Your PR description includes the rationale for setting the `notBefore` to today instead of something closer to the `windowEnd` (trillian with the config we run won't accept future-dated certs)  but it isn't present in this comment. Can you add the larger motivating context here? I think it would be valuable for someone reading the code.